packages/app-desktop/gui/Sidebar/Sidebar.tsx

```diff
@@ -493,6 +493,21 @@ const SidebarComponent = (props: Props) => {
 	};
 
 	const renderAllNotesItem = (theme: Theme, selected: boolean) => {
+		const all_notes_folder_id = 'all_notes_folder_id';
```

Comment:
Please check the coding style - we use camel case for variables and constants.

Reply:
Right got it, this should be a quick fix!